### PR TITLE
Feature/home page tidy up

### PR DESCRIPTION
**Pull request by simaob**
This completes this issue: https://github.com/simaob/posso-ir-api/issues/50

It moves stats into a different route and adds a new layout for the homepage explaining the two key routes to join the backend.

Translations were made using Google Translate, so I can't promise they are great... even less so the slovak one!

Next step will be to add a sign up pages for shopkeepers.

It looks like this:

Screenshot 2020-04-27 at 09 33 57
Screenshot 2020-04-27 at 09 34 11

### CODE REVIEW

**Comment by hugopeixoto** [comment]
"Colaborador", in the portuguese version, sounds too much like "employee", when contrasted with "Lojista" (which can be seen as employer). I would suggest changing it to "Voluntário/a", or something that does not overlap with employment terms.

File: config/locales/en.yml [hunk 1]
```diff
@@ -95,7 +95,24 @@ en:
...
+        store_owner_2: 'Add or link to your store.'
+        store_owner_3: 'Record the status of the queues on the website or in the application.'
+        store_owner_4: 'Help your customers!'
+        register: 'Register'
+        collaborator: 'Collaborator'
+        collaborator_1: 'Sign up to the website.'
+        collaborator_2: 'Add missing stores near you.'
+        collaborator_3: 'Report issues with existing stores.'
+        collaborator_4: 'Help us offer a complete service!'
+        download_the_app: 'The app'
```
Comment (hugopeixoto) [inline config/locales/en.yml:111]:
```suggestion
        download_the_app: 'Download the app'
```

File: config/locales/pt.yml [hunk 2]
```diff
@@ -95,7 +95,24 @@ pt:
...
+        store_owner_2: 'Adicione ou associe-se à sua loja.'
+        store_owner_3: 'Registe o estado das filas pelo site ou na aplicação.'
+        store_owner_4: 'Ajude os seus clientes!'
+        register: 'Registe-se'
+        collaborator: 'Colaborador'
+        collaborator_1: 'Registe-se no site'
+        collaborator_2: 'Adicione lojas que estejam em falta perto de si.'
+        collaborator_3: 'Reporte problemas com lojas registadas no sistema.'
+        collaborator_4: 'Ajude-nos a oferecer um serviço robusto!'
+        download_the_app: 'A aplicação'
```
Comment (hugopeixoto) [inline config/locales/pt.yml:111]:
```suggestion
        download_the_app: 'Descarregue a aplicação'
```

**Comment by simaob** [comment]
Yes, that makes sense! I think I thought about it lightly. I'll change it, though if we could have a neutral gender one, might be better, to avoid sensitivities. Or shall I go with the /a?=D

**Comment by simaob** [comment]
renamed the literals to contributor instead of collaborator, as I don't like it either in english.

**Comment by hugopeixoto** [comment]
> Yes, that makes sense! I think I thought about it lightly. I'll change it, though if we could have a neutral gender one, might be better, to avoid sensitivities. Or shall I go with the /a?=D

I would also prefer a gender neutral term. Do you have anything in mind?

**Comment by simaob** [comment]
Not really:( was trying to look at synonyms and scratching my brain, but no good ideas... maybe "Curador de dados" but too many words.

**Comment by hugopeixoto** [comment]
Maybe instead of nouns `Lojista / Colaborador`, we could have: `Sou lojista / Quero contribuir`. This would avoid the noun all together. Uber does this, using `Receber / Viajar`.

**Comment by simaob** [comment]
I like it! =)

**Comment by simaob** [comment]
I'm trying: "É lojista?" and "Quer contribuir?" , without needing to change the instructions afterwards.

**Comment by hugopeixoto** [comment]
Sounds good!

File: config/locales/en.yml [hunk 3]
```diff
@@ -95,7 +95,24 @@ en:
         page_time: 'Server time'
     home:
       index:
-        title: 'Welcome to Can I Go management system'
+        title: 'Can I Go?'
+        intro_html: 'In order to contribute to reducing the size of queues in stores and thus helping keep social distance, the application <i>Can I Go?</i> was born. See how you can contribute.'
+        store_owner: 'Own a shop?'
+        store_owner_1: 'Sign up to the website and get your store owner code.'
+        store_owner_2: 'Add or link to your store.'
```
Comment (hugopeixoto) [inline config/locales/en.yml:102]: > link to your store

sounds like they will add an URL to their store. Maybe "Add or claim your store"? Not really sure of how this mechanism works to suggest a good verb.

File: config/locales/en.yml [hunk 4]
```diff
@@ -95,7 +95,24 @@ en:
         page_time: 'Server time'
     home:
       index:
-        title: 'Welcome to Can I Go management system'
+        title: 'Can I Go?'
+        intro_html: 'In order to contribute to reducing the size of queues in stores and thus helping keep social distance, the application <i>Can I Go?</i> was born. See how you can contribute.'
+        store_owner: 'Own a shop?'
+        store_owner_1: 'Sign up to the website and get your store owner code.'
```
Comment (hugopeixoto) [inline config/locales/en.yml:101]: > Sign up to the website and get your store owner code
Maybe just "sign up and get your store owner code"?

**Pull request merged**